common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -21,7 +21,7 @@
 
                     @topLevelSection.getEditionalisedNavLinks(edition).map { sectionItem =>
                         <li class="navigation-group__item">
-                            <a href="@LinkTo { @sectionItem.url }">@sectionItem.name</a>
+                            <a href="@LinkTo { @sectionItem.url }" data-link-name="nav : secondary : @sectionItem.name">@sectionItem.name</a>
```

Comment:
I think we can actually make it clearer and call it just the `sectionItem.name`. The rest just adds to the over complicated nature of our current nav. 

Previously people could click on the nav links in the footer and the nav links in the header and we needed to differentiate. But we don't need to do that anymore with the new header so it should be okay to omit this.